src/components/atoms/GpuRangeSliders.tsx

```diff
@@ -28,8 +28,8 @@ const GpuRangeSliders: FC<{ showSeparator: boolean }> = ({ showSeparator }) => {
   );
   const { setMinFreq, setMaxFreq } = useSetGpuFrequency();
 
-  if (!(min && max)) {
-    return <span>Error: Missing GPU Information</span>;
+  if (!(min && max) || currentMin < '1') {
```

Comment:
currentMin can't be compared to a string, I'm assuming you meant to compare it to a number?

Reply:
This is also fixed.